### PR TITLE
Endpoints test

### PR DESCRIPTION
**Pull request by Gayaossi**
The changes made to this page:
Added a button that allows adding a new endpoint.
Adding a button for multiple selection (once you click on it, you can select several endpoints to make changes on them, such as deleting the endpoints or installing chipsec on them).

### CODE REVIEW

**Comment by maziland** [comment]
Change the title of this PR, make it related to the changes

File: controllers/agents.js [hunk 1]
```diff
@@ -1,52 +1,40 @@
...
-const chipsecService = require('../services/chipsec')
 
 async function addNode(req, res) {
     try {
-        //validate hostId
-        const hostId = req.params.hostId
+        const hostId = req.params.hostId;
         const osType = req.params.osType;
         if (!hostId || !osType) {
-            return res.status(400).send({ error: 'hostID or osType parameter is required' });
```
Comment (tal-az) [inline controllers/agents.js]: why give less information in the log?

**Review by tal-az — APPROVED** [review]

File: views/endpoints.ejs [hunk 2]
```diff
@@ -71,23 +87,125 @@
...
+
+        document.querySelectorAll('.install-chipsec').forEach(button => {
+            button.addEventListener('click', () => {
+                const uuid = button.dataset.uuid;
+                console.log('Installing CHIPSEC on endpoint:', uuid);
+                // Implement the actual installation logic here
+            });
+        });
+    });
+</script>
```
Comment (GeneralSnuggles) [inline views/endpoints.ejs:197]: Need to move to .js file in public

Comment (Gayaossi) [reply views/endpoints.ejs:197]: will resolve after POC

**Pull request merged**